resources/lang/pt-br/forum.php

```diff
@@ -47,8 +58,8 @@
         'post_number_input' => 'digite número do post',
         'total_posts' => ':posts_count posts no total',
     ],
-    'subforums' => 'Subfórums',
-    'title' => 'osu!community',
+    'subforums' => 'Sub-fórums',
```

Comment:
Still wrong, haha. Should be "Sub-fóruns" or "Subfóruns". Not an actual word in Portuguese, but it still should respect general grammar rules when borrowed from English.